docs/mmteb/points/768.jsonl

```diff
@@ -0,0 +1 @@
+{"GitHub": "awinml", "New dataset": 8}
```

Comment:
Points Calculation: 4 (for 2 new datasets) + 4 bonus points for the Japanese x Clustering task.